package.json

```diff
@@ -22,6 +22,7 @@
   "author": "Samuel Greene <sgreene@smartprocure.us>",
   "license": "MIT",
   "dependencies": {
+    "bluebird": "^3.5.0",
```

Comment:
we should not take bluebird as a dependency for futil - those methods like async map/reduce/etc we should split out to a separate `bluebird-fp` library